services/graphql-server/src/graphql/utils/retrieve-youtube-videos.js

```diff
@@ -0,0 +1,29 @@
+const { get } = require('@base-cms/object-path');
```

Comment:
You might as well move this logic directly into the resolver file. No need to make it a separate file when it’s only used for resolving the fields.